server/service/devices.go

```diff
@@ -606,6 +606,7 @@ func (r *getDeviceSoftwareRequest) deviceAuthToken() string {
 
 type getDeviceSoftwareResponse struct {
 	Software []*fleet.HostSoftwareWithInstaller `json:"software"`
+	Count    int                                `json:"count"`
```

Comment:
Any particular reason to prefer `int` over `uint` here?

Reply:
I think generally `int` should be used when it's a number in the "mathematical sense" (not an `id`), because if we ever do arithmetics with this `uint` can be a footgun if any of the operations result in a negative value.

Reply:
To illustrate what I mean: https://go.dev/play/p/f-ViA-qstLm